test/library/packages/LinearAlgebra/performance/norm-perf.chpl

```diff
@@ -0,0 +1,105 @@
+use LinearAlgebra, Random;
+use Time;
+use Math;
+
+config const m = 3,
+             iters=10,
+             correctness=false;
+
+config const testMatrixNorm=true;
+
+const nbytes = numBytes(real);
+
+var A: [1..m, 1..m] real;
+var b: [1..m] real;
+
+if(correctness) {
```

Comment:
Normally a correctness test will test the same (or similar) input as the performance test. In this case, the correctness test is pretty much an entirely different test. I think this is acceptable, but not ideal.